hki worksheet.rb

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
In the future you should put your code in the provided files.

Also, it looks like you turned in your solution twice.